DEV.md

```diff
@@ -0,0 +1,143 @@
+# `stablesats`
+- [Dependencies](#dependencies)
+    - [Tools](#tools)
+- [Getting started](#getting-started)
+    - [Local Development Mode](#local-development-mode)
+    - [Staging Development Mode](#staging-development-mode)
+- [How to run stablesats](#how-to-run-stablesats)
```

Comment:
information about staging should now be public.

Reply:
I will update the section on staging.

Reply:
Also removed the section on staging as discussed